tests/update/patch_before_migration.pm

```diff
@@ -40,16 +40,27 @@ sub patching_sle {
 
     if (get_var('MINIMAL_UPDATE')) {
         minimal_patch_system(version_variable => 'HDDVERSION');
+        remove_test_repositories;
     }
 
     if (get_var('FULL_UPDATE')) {
         fully_patch_system();
         type_string "reboot\n";
-        $self->wait_boot(textmode => !is_desktop_installed, ready_time => 600);
-        $self->setup_migration;
+        $self->wait_boot(textmode => !is_desktop_installed(), ready_time => 600);
+        # Go back to the initial state, before the patching
+        $self->setup_migration();
     }
 
-    de_register(version_variable => 'HDDVERSION');
+    if (get_var('FLAVOR', '') eq 'Server-DVD-Updates') {
```

Comment:
this is check_var recoded.

But I wonder if this FLAVOR check will scale. Maintenance project managers want incidents to be tested - and they have a different flavor. So possibly we need a different variable they have in common?